.github/workflows/prepare-release.yml

```
@@ -33,3 +35,58 @@ jobs:
           publish: "npm run release"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  release:
+    name: Deploy and release to sandbox
+    needs: changesets
+    if: needs.prepare.outputs.published == 'true'
+    runs-on: ubuntu-latest
```

I recommend freezing the ubuntu version so it doesn't suddenly stop working
```suggestion
    runs-on: ubuntu-22.04
```